src/uiSelectController.js

```diff
@@ -557,12 +558,12 @@ uis.controller('uiSelectCtrl',
             return false;
           }
           var inputWidth = containerWidth - input.offsetLeft - 10;
-          if (inputWidth < 50) inputWidth = containerWidth;
+          if (inputWidth < minInputWidth) inputWidth = containerWidth;
           ctrl.searchInput.css('width', inputWidth+'px');
           return true;
         };
 
-    ctrl.searchInput.css('width', '10px');
+    ctrl.searchInput.css('width', minInputWidth+'px');
```

Comment:
Although this number might not be accounting for some padding that's added when calling `updateIfVisible`, so some tweaks might be necessary...